testssl.sh

```diff
@@ -15879,9 +16042,11 @@ run_sweet32() {
      if [[ $sclient_success -eq 0 ]] && "$ssl2_sweet" ; then
           pr_svrty_low "VULNERABLE"; out ", uses 64 bit block ciphers for SSLv2 and above"
           fileout "SWEET32" "LOW" "uses 64 bit block ciphers for SSLv2 and above" "$cve" "$cwe" "$hint"
+          "$tls1_1_vulnable" && set_grade_cap "C" "Uses 64 bit block ciphers with TLS1.1+ (vulnerable to SWEET32)"
      elif [[ $sclient_success -eq 0 ]]; then
           pr_svrty_low "VULNERABLE"; out ", uses 64 bit block ciphers"
           fileout "SWEET32" "LOW" "uses 64 bit block ciphers" "$cve" "$cwe" "$hint"
+          "$tls1_1_vulnable" && set_grade_cap "C" "Uses 64 bit block ciphers with TLS1.1+ (vulnerable to SWEET32)"
```

Comment:
Shouldn't this be just TLS1.1?  ``$tls1_1_vulnable` is only set to true if TLS 1.1 has a 64 bit cipher. TLS 1.2 shouldn't have any 64 bit cipher.

Second remark: why do you limit this to TLS 1.1 in the first place?

Reply:
Second point in this: https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide#changes-in-2009n-19-january-2017---3-april-2017

Should be only TLS1.1 yes, and not TLS1.1 (since TLS1.2 doesn't have)

Reply:
2nd point: ok, read that. Thanks. However it's hard to believe they don't penalize it for earlier versions. This needs to be checked maybe with a real life example. I add that as an open point